internal/wasm/func_validation_test.go

```diff
@@ -2709,8 +2709,7 @@ func TestModule_funcValidation_SIMD(t *testing.T) {
 	vi2v := func(vec OpcodeVec) (ret []byte) {
 		ret = addV128Const(ret)
 		return append(ret,
-			OpcodeI32Const,
-			1, 1, 1, 1,
+			OpcodeI32Const, 1,
```

Comment:
needs leb128 encoding but yeah somehow this worked since before but fixed it 😄

Reply:
ah cool